tools/cldr-apps/js/src/esm/cldrCoverageReset.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * cldrCoverageReset: encapsulate Survey Tool code for auto-resetting coverage
+ */
+import * as cldrCoverage from "../esm/cldrCoverage.js";
+import * as cldrGui from "../esm/cldrGui.js";
+import * as cldrLoad from "../esm/cldrLoad.js";
+import * as cldrMenu from "../esm/cldrMenu.js";
+import * as cldrText from "../esm/cldrText.js";
+
+import { notification } from "ant-design-vue";
+
+const TEST_DANGER = false && cldrStatus.getIsUnofficial(); // false for production, true only when testing
```

Comment:
@srl295 I accepted your suggestion here -- not likely we would leave it true, still it's good to be cautious!